### ISSUE TITLE
Release 1.3.3 does not contain a bug-fixing commit (d8203ba94d528b092e0ff5a52a51af28f8f592f1)

### ISSUE THREAD

**Issue by sudoforge** [mumble-voip/mumble#4728]
**Describe the bug**

When setting a superuser passphrase using the `-supw` argument, the server sets the passphrase successfully, but erroneously exits with a status of `1`.

**Steps to Reproduce**

1. Download the 1.3.3 static binary
2. Run `/path/to/server_binary.x86 -supw foo && echo $?`

**Expected behavior**
The server should start up, set the passphrase, output an informational message, and exit 0.

**Desktop (please complete the following information):**
 - OS: 64-bit Linux (5.10.11 kernel, but this doesn't matter)
 - Version: mumble-server 1.3.3

**Additional context**
This has been previously reported in #3911 and subsequently fixed in #3998. On `HEAD`, the fix commit is d8203ba94d528b092e0ff5a52a51af28f8f592f1, and the (empty) merge commit is f8ee53688353c8f5e1650504a961ee582ac16668. This was merged 11 months ago (about 7 months prior to the 1.3.3 release on Oct 4).

The 1.3.3 tree does not contain this commit, or this fix in another commit. We can confirm this by checking for both the sha itself and more generally on the message, or specifically with a regex. These are documented below:

```
# Check to see if the release tree contains the commit by sha
$ git tag --contains d8203ba94d528b092e0ff5a52a51af28f8f592f1
<no matches, empty output>

# ... or by commit message (based on the subject)
# this is useful to find the commit in the event it was cherry-picked or otherwise had a re-computed sha on the tree
$ git log --grep "$(git log -1 --format='%s' d8203ba94d528b092e0ff5a52a51af28f8f592f1)" refs/tags/1.3.3

# ... or by regular expression matching on the changed line
$ git log -G'qInfo\("Superuser password set on server' refs/tags/1.3.3
```

**Comment by sudoforge** [comment]
I talked about this in the development IRC channel, and @Krzmbrzl added the `backport-needed` label to #3998. It should therefore be included in the 1.3.4 release.

**Comment by sudoforge** [comment]
The fix has been cherry picked into the 1.3.4 branch, and this issue can now be closed.

```
➜ git log --grep "$(git log -1 --format='%s' d8203ba94d528b092e0ff5a52a51af28f8f592f1)" refs/tags/1.3.4
commit 9fc1c188cf23612e0a2083af20735c1ee1571a07
Author: Robert Adam <dev@robert-adam.de>
Date:   Wed Mar 18 12:52:20 2020 +0100

    src/murmur/main.cpp: Use qInfo instead of qFatal for normal logging.

    For some reason qFatal has been used to log information such as the
    version requested by the --version argument.
    abdb5004bb92b28271673e33df9cc3e314f5711a made qFatal exit with an
    exit status of one leading to successful operations that happened
    to contain some print-out to leave an exit status of 1 (#3911).
```

This was further verified by inspecting commit 9fc1c188cf23612e0a2083af20735c1ee1571a07 manually to ensure it contained the appropriate delta.
